exporter/loggingexporter/logging_exporter.go

```diff
@@ -24,6 +24,10 @@ import (
 	"github.com/open-telemetry/opentelemetry-service/exporter/exporterhelper"
 )
 
+type loggingExporter struct {
```

Comment:
IMO this struct is not needed because we still need to pass the same `*zap.Logger` to `NewTraceExporter` and `NewMetricsExporter`.